### DISCUSSION TITLE
My replies show up in some "conversations" but not others.

### DISCUSSION THREAD

**Discussion by GrantEdwards** [protz/thunderbird-conversations#1828] [General]
In some of the "conversations" in my inbox my replies are show as expected (with a tag "In Sent Items"). In other conversations in my inbox, none of my replies show up in the "conversation" — I have to browse to "Sent Items" to see my replies.

Any ideas what might cause this?

**Comment by GrantEdwards** [comment]
I run Thunderbird on two machines, both of which connect to the same Exchange server. All e-mails sent by me from either machine show up in the "Sent" folder on both machines. However, in the conversation view of my inbox, only e-mails sent by me **_from the current machine_** show up.

How do I get conversation view of inbox to include all of the emails in the "Sent" folder?

**Reply by Standard8** [reply]
If you're on a POP account, that might not be possible. If you're on IMAP, then in theory Thunderbird should at some stage download the email in the sent folder, and then index it - at which point Conversations would let it show up.

It might help to try this: Right-clicking on the folder, selecting properties, and then turning on "When getting new messages for this account, always check this folder".

If not, then really its a Thunderbird issue that you'd need to bring up on https://support.mozilla.org/products/thunderbird

**Reply by GrantEdwards** [reply]
I'm using the OWL plugin with Exchange Server OWA access.

I've checked the Sent folder's "When getting new messages for this account, always check this folder" box, clicked "Get Messages" (for both inbox and Sent), and it didn't seem to help. Maybe it will help going forward.

Thanks for the suggestion.

**Comment by GrantEdwards** [comment]
I think that checking the "When getting new messages for this account, always check this folder" box did fix the problem. It didn't happen right away, but after doing some other stuff (including restarting Thunderbird), the "missing" messages have now shown up in the inbox conversation threads as desired. Yay!

I'm a bit confused what "check this folder" actually means. Without the box checked, the folder message listing always showed up correctly, and I could open and read the messages without any problem, so how was I to know that the folder wasn't being "checked"  when it needed to be "checked"?

[irrelevant-whine]
This seems to be another case of overly vague language in the UI. Earlier today I had a fight with the error message "can not talk to server". It would have saved a lot of time if that error message actually stated the failure — e.g. can't resolve hostname, no route to host, TCP connection timed out, TCP connection refused, SSL handshake failed, server's SSL certificate invalid, user authentication failed, server not responding to requests, etc. [it turns out it was a certificate problem.] Maybe there's a log somewhere containing useful failure info, but I couldn't find it.
[/irrelevant-whine]
